ntc_templates/templates/alcatel_sros_show_port_description.textfsm

```diff
@@ -0,0 +1,22 @@
+Value PORT_ID (\S+)
```

Comment:
In case it's useful.
If you like this suggestion, click the Commit suggestion button to accept my edit. Thank you!

```suggestion
Value PORT_ID ((\d+\/?)+)
```

Reply:
Gah, the `A/1` port isn't going to appreciate my pattern. Ugh.

Reply:
@bennnnnnnn @evilmonkey19
I've updated the pattern to accept an optional alphabetical character as part of the port name. (ex: `A/1`)

Please take a look. Thank you.